ports/lisathetimeless/port.json

```diff
@@ -0,0 +1,28 @@
+{
+    "version": 2,
+    "name": "lisathetimeless.zip",
+    "items": [
+        "LISA The Timeless.sh",
+        "lisatheTimeless"
```

Comment:
foldernames need to be complety lowercase
"lisatheTimeless"

vs 
""lisathetimeless"